.github/workflows/smoke_test.yml

```diff
@@ -0,0 +1,41 @@
+name: Smoke Tests
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+defaults:
+  run:
+    working-directory: typescript
+jobs:
+  test:
+    name: Smoke Tests
+    timeout-minutes: 15
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      
+      - name: Use Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
```

Comment:
You need version >= 18. I've experienced some instabilities with version 18 in the past, but let's try